{"name":"logIn_Test_success","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [false] but found [true]","trace":"java.lang.AssertionError: expected [false] but found [true]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertFalse(Assert.java:78)\r\n\tat org.testng.Assert.assertFalse(Assert.java:88)\r\n\tat DemeGuru99.tests.LogInTest.logIn_Test_success(LogInTest.java:11)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:806)\r\n\tat org.testng.TestRunner.run(TestRunner.java:601)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:433)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Page screenshot","source":"ddba5850-ade3-4803-8957-33e0714a3883-attachment","type":"image/png"},{"name":"logIn_Test_success failed and screenshot taken!","source":"b18a0ae7-c50b-4ae5-98df-3a9735d7116a-attachment","type":"text/plain"}],"parameters":[],"start":1665997957899,"stop":1665997958308,"uuid":"139dc103-c27e-403f-a2aa-71a79a24c8ed","historyId":"b7f64d28472e2f7834f08bc18af45c81","fullName":"DemeGuru99.tests.LogInTest.logIn_Test_success","labels":[{"name":"package","value":"DemeGuru99.tests.LogInTest"},{"name":"testClass","value":"DemeGuru99.tests.LogInTest"},{"name":"testMethod","value":"logIn_Test_success"},{"name":"parentSuite","value":"Page Object test example"},{"name":"suite","value":"TEST DEMOGURU99.COM"},{"name":"subSuite","value":"DemeGuru99.tests.LogInTest"},{"name":"host","value":"LVN01000136"},{"name":"thread","value":"21980@LVN01000136.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}